ui/components/component-library/button-base/button-base.stories.tsx

```diff
@@ -160,35 +164,35 @@ ExternalLink.args = {
   externalLink: true,
 };
 
-export const Disabled = (args) => (
+export const Disabled: ComponentStory<typeof ButtonBase> = (args) => (
   <ButtonBase {...args}>Disabled Button</ButtonBase>
 );
 
 Disabled.args = {
   disabled: true,
 };
 
-export const Loading = (args) => (
+export const Loading: ComponentStory<typeof ButtonBase> = (args) => (
   <ButtonBase {...args}>Loading Button</ButtonBase>
 );
 
 Loading.args = {
   loading: true,
 };
```

Comment:
This story seems to be broken?

<img width="608" alt="Screenshot 2023-05-05 at 10 28 19 AM" src="https://user-images.githubusercontent.com/8112138/236359009-e90aee65-a1be-4ef1-b594-7c9d2b6b572a.png">

Checking it against `develop` branch storybook hosted here https://metamask.github.io/metamask-storybook/?path=/story/components-componentlibrary-buttonbase--loading

This is why it's great to add a before and after screencast of storybook to catch any visual regressions